src/twisted/web/http.py

```diff
@@ -1386,7 +1386,7 @@ def getClientIP(self):
         @returns: the client IP address
         @rtype: C{str}
         """
-        if isinstance(self.client, address.IPv4Address):
+        if isinstance(self.client, (address.IPv4Address, address.IPv6Address)):
```

Comment:
This is fine... but maybe we should use  `_IPAddress` here... I don't know which is better.

Reply:
I'm reluctant to use a private implementation even though this is inside Twisted.